SU2_CFD/src/variables/CNEMOEulerVariable.cpp

```diff
@@ -168,14 +166,9 @@ CNEMOEulerVariable::CNEMOEulerVariable(su2double val_pressure,
 
     Solution(iPoint,nSpecies+nDim)       = rho*(energies[0]+0.5*sqvel);
     Solution(iPoint,nSpecies+nDim+1)     = rho*(energies[1]);
-
-    Solution_Old = Solution;
```

Comment:
If you suspect this may be happening somewhere else I can tell you how to instrument the code to find it.

Reply:
Thanks Pedro, I still need to check what is happening at each iteration, but first wanted to sort out the initialization issue